Tests/Marketplace/zip_packs.py

```diff
@@ -212,12 +229,13 @@ def main():
     storage_bucket_name = option.bucket_name
     zip_path = option.zip_path
     artifacts_path = option.artifacts_path
```

Comment:
I don't see where you are using this script argument. You'll want to use it since the artifacts path is different for GitLab and CircleCI.

Reply:
It's used in lines 286 and 288